net.sourceforge.Artha.yaml

```diff
@@ -0,0 +1,140 @@
+command: artha
+
+rename-desktop-file: artha.desktop
+#rename-icon: artha.png
+
+finish-args:
+  - --socket=x11
+  - --device=dri
+  # dbus session bus for artha
+  - --own-name=org.artha.unique
+  
+modules:
+
+- shared-modules/gtk2/gtk2.json 
+    
+# Wordnet 3.0 Needs tcl 8.5.x 
+- name: tcl8.5.19
+  cleanup: 
+    - /cache
+    - /man
+    - /share/aclocal
+    - /share/devhelp
+    - /include
+    - /lib/pkgconfig
+    - /share/gtk-doc
+    - /share/man
+    - /lib/*.a
+    - '*.a'
+    - '*.la'
+  sources:
+  - type: archive
+    url: ftp://ftp.tcl.tk/pub/tcl/tcl8_5/tcl8.5.19-src.tar.gz
+    sha256: d3f04456da873d17f02efc30734b0300fb6c3b85028d445fe284b83253a6db18
+  subdir: unix
+  build-options:
+    no-debuginfo: true
+  post-install:
+  - make install-private-headers 
+  - ln -v -sf tclsh8.6 ${FLATPAK_DEST}/bin/tclsh 
+  - chmod -v 755 ${FLATPAK_DEST}/lib/libtcl8.5.so
+
+# Wordnet 3.0 Needs tk 8.5.x
+- name: tk8.5.19
+  cleanup: 
+    - /cache
+    - /man
+    - /share/aclocal
+    - /share/devhelp
+    - /include
+    - /lib/pkgconfig
+    - /share/gtk-doc
+    - /share/man
+    - /lib/*.a
+    - '*.a'
+    - '*.la'
+  sources:
+  - type: archive
+    url: ftp://ftp.tcl.tk/pub/tcl/tcl8_5/tk8.5.19-src.tar.gz
+    sha256: 407af1de167477d598bd6166d84459a3bdccc2fb349360706154e646a9620ffa
+  subdir: unix
+  post-install:
+  - make install-private-headers 
+  - ln -v -sf wish8.6 ${FLATPAK_DEST}/bin/wish 
+  - chmod -v 755 ${FLATPAK_DEST}/lib/libtk8.5.so
+
+- name: wordnet
+  cleanup: 
+    - /cache
+    - /man
+    - /share/aclocal
+    - /share/devhelp
+    - /include
+    - /lib/pkgconfig
+    - /share/gtk-doc
+    - /share/man
+    - /lib/*.a
+    - '*.a'
+    - '*.la'
+  buildsystem: autotools
+  config-opts:
+  - --with-tcl=/app/lib
+  - --with-tk=/app/lib
+  - --enable-static=no 
+  sources:
+  - type: archive
+    url: http://wordnetcode.princeton.edu/3.0/WordNet-3.0.tar.bz2
+    sha256: 6c492d0c7b4a40e7674d088191d3aa11f373bb1da60762e098b8ee2dda96ef22
+    patch: 
+    
+- name: dbus-glib
+  cleanup: 
+    - /cache
+    - /man
+    - /share/aclocal
+    - /share/devhelp
+    - /include
+    - /lib/pkgconfig
+    - /share/gtk-doc
+    - /share/man
+    - /lib/*.a
+    - '*.a'
+    - '*.la'
+  buildsystem: autotools
+  sources:
+  - type: archive
+    url: https://dbus.freedesktop.org/releases/dbus-glib/dbus-glib-0.112.tar.gz
+    sha256: 7d550dccdfcd286e33895501829ed971eeb65c614e73aadb4a08aeef719b143a
+
+
+- name: artha
+  cleanup: 
+    - /cache
+    - /man
+    - /share/aclocal
+    - /share/devhelp
+    - /include
+    - /lib/pkgconfig
+    - /share/gtk-doc
+    - /share/man
+    - /lib/*.a
+    - '*.a'
+    - '*.la'
+  buildsystem: autotools
+  sources:
+  - type: archive
+    url: https://sourceforge.net/projects/artha/files/artha/1.0.5/artha-1.0.5.tar.bz2
+    sha256: 5a0d568a5d5335f98b3a04180ec28de6ce280aa68d21036601e7f832dd3e990c
+  post-install:
+  - mkdir -p ${FLATPAK_DEST}/share/icons/hicolor/{48x48,128x128}/apps/
```

Comment:
```suggestion
```

`install -D` does that for you

Reply:
it does not create 2 directories at once. tried that already but fails. Hence creating 2 directories with mkdir instead.

Reply:
line 136 and 138 WILL create the directories